wgs_somatic-run-wrapper.py

```diff
@@ -131,16 +133,35 @@ def wrapper():
                     n = k
                     n_ID = [val for val in pair_dict_all_pairs.get(k) if val != 'normal'][0]
```

Comment:
This looks a bit odd. You use a list comprehension but only ever use the first value of it anyway via `[0]`.

Reply:
It is because the list has two values: "tumor" or "normal" and "tumor ID". So I want to find the tumor ID if "normal" is in the list so that's why I remove "normal" and then use the value that is left in the list. Since the list now only has one value, I extract it with [0]. But yeah, this should probably be solved in a better way.